src/App.tsx

```diff
@@ -1,13 +1,14 @@
 import { useEffect, useState } from "react";
 import { App as AntdApp, Layout, Row, Col, Collapse, theme } from "antd";
-import Navbar from "./Navbar";
+import Navbar from "./components/Navbar";
 import AgreementData from "./AgreementData";
 import AgreementHtml from "./AgreementHtml";
 import Errors from "./Errors";
 import TemplateMarkdown from "./TemplateMarkdown";
 import TemplateModel from "./TemplateModel";
 import useAppStore from "./store";
 import SampleDropdown from "./SampleDropdown";
+import CustomFooter from "./components/Footer";
```

Comment:
nit: Can we please have both folder name and component name to be the same?
For me `Footer` sounds more appropriate.

Reply:
Okay, sure would look into that